pytest/tests/sanity/rpc_key_value_changes.py

```diff
@@ -249,12 +349,33 @@ def set_value_1():
 function_call_1_thread.join()
 
 tx_block_hash = function_call_2_response['result']['transaction_outcome']['block_hash']
-tx_account_id = nodes[0].signer_key.account_id
 
-state_changes_request = {
+state_changes_in_block_request = {
     "block_id": tx_block_hash,
+}
+
+expected_changes_in_block_response = {
+    "block_hash": state_changes_in_block_request['block_id'],
+    "changes": [
+        {
+            "type": "account_touched",
+            "account_id": contract_key.account_id,
+        },
+        {
+            "type": "data_touched",
+            "account_id": contract_key.account_id,
+        },
+        {
+            "type": "access_key_touched",
+            "account_id": contract_key.account_id,
+        }
+    ]
+}
```

Comment:
FYI, I am yet to learn why "access_key_touched" happens.

Context: The test calls the same smart contract concurrently, so two transactions land to the same block changing the same key. The account is touched since it gets charged, but why is there a change to the access key? :man_shrugging:

Reply:
Probably because nonce is incremented?